components/formats-api/src/loci/formats/meta/MetadataConverter.java

```diff
@@ -884,7 +884,7 @@ private static void convertImages(MetadataRetrieve src, MetadataStore dest,
       catch (NullPointerException e) { }
 
       try {
-        List<MapPair> map = src.getImagingEnvironmentMap(i);
+        java.util.List<ome.xml.model.MapPair> map = src.getImagingEnvironmentMap(i);
```

Comment:
This could be reduced to `List<MapPair>` (i.e. resulting in no change in this file)